phpBB/config/default/container/services_files.yml

```diff
@@ -1,4 +1,35 @@
 services:
+    attachment.delete:
```

Comment:
I guess moving the attachment classes to services_attachments.yml sounds like a reasonable way of handling this?